package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "good-graylog2",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "test": "./node_modules/.bin/gulp test"
+  },
+  "engines": {
+    "node": ">=0.10.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/CascadeEnergy/good-graylog2.git"
+  },
+  "author": {
+    "name": "CascadeEnergy",
+    "email": "chris.cortez@cascadeenergy.com"
+  },
+  "contributors": [
+    {
+      "name": "Nick Warlen",
+      "email": "nick.warlen@cascadeenergy.com"
+    }
+  ],
+  "license": "MIT",
```

Comment:
We should add a `keywords` array. So people can find the module. Things like 'good', 'hapi', 'hapijs', 'graylog', 'graylog2', 'logging', etc whatever you think of.